$OpenBSD: patch-src_nvim_CMakeLists_txt,v 1.3 2020/08/18 19:15:54 edd Exp $

No need for libc++abi, only available on clang archs and only needed for
luajit (we use lua instead).

Index: src/nvim/CMakeLists.txt
--- src/nvim/CMakeLists.txt.orig
+++ src/nvim/CMakeLists.txt
@@ -366,10 +366,6 @@ endforeach()
 
 # Our dependencies come first.
 
-if (CMAKE_SYSTEM_NAME MATCHES "OpenBSD")
-  list(APPEND NVIM_LINK_LIBRARIES pthread c++abi)
-endif()
-
 if (LibIntl_FOUND)
   list(APPEND NVIM_LINK_LIBRARIES ${LibIntl_LIBRARY})
 endif()
